daprdocs/content/en/getting-started/quickstarts/resiliency/resiliency-state-quickstart.md

```diff
@@ -0,0 +1,900 @@
+---
+type: docs
+title: "Quickstart: Service-to-component resiliency"
+linkTitle: "Resiliency: State Management"
```

Comment:
I'd recommend rephrasing this to resiliency: app to service

Reply:
@greenie-msft should I also change the `title` to "Quickstart: App-to-service resiliency"?

Reply:
ive changed both